### ISSUE TITLE
Weird crash

### ISSUE THREAD

**Issue by rootapprox** [eh2k/squares-and-circles#107]
![photo_2024-11-16_18-47-31](https://github.com/user-attachments/assets/179f2e43-e81c-4d6f-b20e-c223fe71c32f)

**Comment by rootapprox** [comment]
also i cant load any of the Patches i have in the patch Storage, always reboots ...

**Comment by rootapprox** [comment]
also unable to restore saved patches .. always takes me to the actual patch..

**Comment by eh2k** [comment]
@rootapprox 

i think there are two issues here, one is this https://github.com/eh2k/squares-and-circles/issues/97 (patch restore - reboot without crashreport).

The other should be the crash (as shot in your crashreport).
In the crash report I can see exactly which engines you used. Unfortunately I have not been able to reproduce it - according to the code-address it should have crashed in the IO-page.

Can you actually reproduce the crash reliably?

**Comment by rootapprox** [comment]
Yes , i can make the crach everytime i want to load a saved patch , it always takes me to the actual patch,
